app/views/layouts/application.html.erb

```diff
@@ -52,7 +52,7 @@
       window.agentPaths = {};
       window.agentNames = [];
       <% if current_user.present? -%>
-        var myAgents = <%= Utils.jsonify(current_user.agents.pluck(:name, :id).inject({}) {|m, a| m[a.first] = agent_path(a.last); m }) %>;
+        var myAgents = <%= Utils.jsonify(current_user.agents.pluck(:name, :id).inject({}) {|m, a| next if a.last.nil?; m[a.first] = agent_path(a.last); m }) %>;
```

Comment:
Oy, that could definitely break things where `pluck` is being used with the specific intention of hitting the DB instead of a cache.  I don't know if we ever do this, but I think I have on other projects.